pandas/util/testing.py

```diff
@@ -1314,13 +1314,13 @@ def assert_frame_equal(left, right, check_dtype=True,
     right : DataFrame
     check_dtype : bool, default True
         Whether to check the DataFrame dtype is identical.
-    check_index_type : bool / string {'equiv'}, default False
+    check_index_type : bool / string {'equiv'}, default 'equiv'
```

Comment:
The type here seems rather verbose. Wondering if it's better served as just `bool or 'equiv'` since 'equiv' appears to be the only allowable string. @datapythonista